deploy/.env

```diff
@@ -1,5 +1,5 @@
 #INTERNAL_HOST_PORT=1235
 #PUBLIC_ORIGIN=http://localhost
 #SPIPES_ENGINE_URL=http://host.docker.internal:8080/s-pipes
-#SPIPES_EDITOR_REST=s-pipes-editor-rest:18115
+#SPIPES_EDITOR_REST_URL=http://localhost:1235/rest
```

Comment:
```suggestion
#S_PIPES_EDITOR_REST_URL=http://localhost:1235/rest
```

Reply:
- You have SPIPES here but S_PIPES above. I suggest to unify it, i.e. look for all SPIPES and replace with S_PIPES, **not only here**